docs/identity/saas-apps/guru-tutorial.md

```diff
@@ -165,4 +165,6 @@ In this section, you test your Microsoft Entra single sign-on configuration with
 
 ## Next Steps
 
-Once you configure Guru you can enforce session control, which protects exfiltration and infiltration of your organization's sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).
\ No newline at end of file
+Once you configure Guru you can enforce session control, which protects exfiltration and infiltration of your organization's sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).
+
+For any additional information, please visit [Guru's Help Center](https://help.getguru.com/s/)
```

Comment:
I forgot to add a period to the end of this sentence - are you able to add one please?